src/NJsonSchema/Generation/SampleJsonDataGeneratorSettings.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace NJsonSchema.Generation
+{
+    /// <summary> Settings for generating sample json data.</summary>
+    public class SampleJsonDataGeneratorSettings
+    {
+        /// <summary>Gets or sets a value indicating whether to generate to generate optional properties. (default: true).</summary>
```

Comment:
`Gets or sets a value indicating whether to generate optional properties (default: true).`